types/raidgroupconfig.go

```diff
@@ -23,7 +23,7 @@ import "github.com/pkg/errors"
 type RaidGroupConfig struct {
 	// Type is the raid group type
 	// Supported values are : stripe, mirror, raidz and raidz2
-	Type PoolRAIDType `json:"type"`
+	Type PoolRAIDType `json:"raidType"`
```

Comment:
Please rename the field `Type` to `RAIDType` as well

Reply:
Sure, have renamed.